almalinux_build_node.py

```diff
@@ -157,13 +169,25 @@ def sigusr_handler(signum, frame):
 
     node_globals.init_supervisors(config)
     builders = []
+    task_queue = queue.Queue(maxsize=config.queue_size)
+
     for i in range(0, config.threads_count):
-        builder = BuildNodeBuilder(config, i, node_terminated,
-                                   node_graceful_terminated)
+        builder = BuildNodeBuilder(
+            config,
+            i,
```

Comment:
Using single char variable names when iterating is fine, however, in this particular use case and for readability purposes, I'd call it `thread_num` or something like that.